workers/data_refinery_workers/processors/test_transcriptome_index.py

```diff
@@ -11,6 +11,7 @@
     ProcessorJobOriginalFileAssociation,
     ProcessorJob
 )
+from unittest.mock import patch
```

Comment:
For AWS testing, I suggest that we maybe try to use the [placebo](https://github.com/garnaat/placebo) library rather than simply mocking calls out.